CHANGES.md

```diff
@@ -1,3 +1,22 @@
+## edge-19.1.3
+
+* Controller
+  * Added `linkerd install-sp` command to install service profiles into the
+    linkerd control plane
```

Comment:
capitalize `Linkerd control plane`

Reply:
👍 to @siggy's capitalization suggestion here.

Reply:
yeah! sorry it got lost in the merge conflict resolving